cgi-bin/DW/Controller/MassPrivacy.pm

```diff
@@ -0,0 +1,159 @@
+   #!/usr/bin/perl
```

Comment:
Can we unindent this file back to 0?

Also we need to add the normal copyright header?